apps/src/tutorialExplorer/roboticsButton.jsx

```diff
@@ -28,14 +45,23 @@ const RoboticsButton = React.createClass({
     };
 
     return (
-      <div style={{float:"left"}}>
+      <div>
         <div style={{display: getResponsiveValue({md: "block", xs: "none"})}}>
-          <a href="/learn/robotics">
-            <img src="/images/learn/robotics-link.png" style={styles.roboticsButtonImage}/>
-          </a>
+          <div style={styles.button}>
+            <a href="/learn/robotics">
+              <div style={{position: "relative"}}>
+                <img src="/images/learn/robotics-link.png" style={styles.roboticsButtonImage}/>
+                <div style={styles.roboticsButtonText}>
+                  {i18n.roboticsButtonText()}
+                  &nbsp;
+                  <i className="fa fa-arrow-right" aria-hidden={true}/>
+                </div>
+              </div>
+            </a>
+          </div>
         </div>
         <div style={roboticsTextStyle}>
-          {i18n.roboticsButtonText({robotics_url: "/learn/robotics"})}
+          <div dangerouslySetInnerHTML={{__html: i18n.roboticsText({robotics_url: "/learn/robotics"})}}/>
```

Comment:
Instead of having HTML in this string can we change the string to "Got robots? View robotics activities and make a tangible Hour of Code for students of any age!" and have the entire div clickable?